Rossum/index.js

```diff
@@ -0,0 +1,63 @@
+
+const Box = require('./Helpers/box.js')
+const Rossum = require('./Helpers/rossum.js');
+
+/**
+ * This is the main function that the Lambda will call when invoked.
+ */
+exports.handler = async (triggeredEvent, context, callback) => {
+  console.log('Event received. Huzzah!');
+  
+  if (isValidEvent(triggeredEvent)) {
+    await processEvent(triggeredEvent, callback);
+  } else {
+    console.log('Invalid event');
+    callback(null, { statusCode: 200, body: 'Event received but invalid' });
+  }
+};
+
+function isValidEvent(triggeredEvent) {
+  return triggeredEvent.body
+};
+
+async function processEvent(triggeredEvent, finalCallback) {
+  const { body } = triggeredEvent;
+  const box = new Box(body);
+
+  try {
+      const containsMetadata = await box.containsSkillsMetadata() // Check if the file already has a Skills card
```

Comment:
This call or function is not needed, you don't need to delete existing cards. The skill-kit does it by itself.